docs/manual/scala/guide/devmode/ServicePort.md

```diff
@@ -0,0 +1,49 @@
+# How are ports assigned to services?
+
+When inspecting the list of running services, you may wonder how ports are being assigned. One thing you should notice is that ports are assigned consistently, meaning that each service will get the same port assigned. This is truly useful, as it allows to write scripts that exercise some service's functionality, and even share the created scripts with the rest of your team. Indeed, the same port is deterministically selected even on different machines!
+
+The algorithm used for assigning a port to each service works as follows:
+
+1) The project's name is hashed.
+2) The hash absolute value is projected on the port range (the default port range is `[49152,65535]`).
+3) The selected port is assigned to the project, if there is no other project claiming the same port. If two or more projects are projected onto the same port, the conflicting projects are alphabetically sorted, and the project that comes first will get the expected port assigned to it. While, the remaining projects will get assigned the closest (strictly increasing) available neighbour port.
+
+In general, you don't need to worry about these details, as in most cases the port range is wide enough to make collisions unlikely. However, there are times when you may still prefer to assign a specific port to a service (for instance, if the automatically assigned port is already is use in your system). You can do so by manually providing a port number for the project's service port setting.
+
+In Maven, you can do this by modifying the service implementations pom configuration:
+
+```xml
+<plugin>
+    <groupId>com.lightbend.lagom</groupId>
+    <artifactId>lagom-maven-plugin</artifactId>
+    <configuration>
+        <servicePort>11000</servicePort>
+    </configuration>
+</plugin>
+```
```

Comment:
Maybe not for this PR, but it seems that we have a few files in the Scala documentation referring to maven. 

Although it's not illegal to build Scala with Maven, it's very unlikely that a team using Scala also chooses for Maven. I suggest that we review this approach and remove references to Maven on Scala documentation. WDYT?